great-idea-website/css/index.css

```diff
@@ -63,4 +63,122 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+hr {
+	width: 80%;
+}
+.main-content {
+	margin: 0 20%;
+}
+
+nav {
+	width: 70%;
+	display: inline-block;
+	margin: 0 0 0 5%;
+
+}
+nav a {
+	text-decoration: none;
+	display: inline-block;
+	margin: 5%;
+}
+
+.logo {
+	display: inline-block;
+}
+
+.top-feature {
+	margin: 5% 10%;
+}
+
+.feature-text {
+	display: inline-block;
+	width: 30%;
+	height: 220px;
+	margin: 5% 10%;
+}
+
+.feature-text h2 {
+	font-size: 70px;
+}
+
+.first-h2 {
+	text-align: center;
+}
+
+.second-h2 {
+	text-align: center;
+}
+
+.third-h2 {
+	text-align: center;
+}
+
+.get-started-btn {
+	margin-left: 25%;
+	width: 50%;
+	height: 15%;
+	background-color: #fff;
```

Comment:
The button in the design file is pretty flat, while the default button has this 3D shadow effect thing going on. To make the button better match the design, you can use `border` to put in your own border for this button.